
Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Importing simConfig from ../data/batch_2023-12-08/batch_2023-12-08_4_4_0_16_cfg.json
Loading file ../data/batch_2023-12-08/batch_2023-12-08_4_4_0_16_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/batch_2023-12-08/batch_2023-12-08_netParams.py

Start time:  2023-12-08 12:44:24.259165

Creating network of 8 cell populations on 1 hosts...
  Number of cells on node 0: 700 
  Done; cell creation time = 0.06 s.
Making connections...
  Distributing synapses based on subcellular connectivity rules...
  Number of connections on node 0: 0 
  Done; cell connection time = 0.07 s.
Adding stims...
  Number of stims on node 0: 0 
  Done; cell stims creation time = 0.00 s.
Recording 0 traces of 0 types on node 0
{cellType: 'PT', endTime: 1000, mech: 'hd', newFactor: 1.0, origFactor: 0.75, property: 'gbar', startTime: 500}
0.0s

Running with interval func using NEURON for 1000.0 ms...
0.1s
0.2s
0.3s
0.4s
0.5s
0.6s
0.7s
0.8s
0.9s
1.0s
   Modifying PT hd gbar by a factor of 0.750000
  Done; run time = 0.23 s; real-time ratio: 4.43.

Gathering data...
  Done; gather time = 0.01 s.

Analyzing...
  Cells: 700
  Connections: 0 (0.00 per cell)
  Spikes: 1348 (1.93 Hz)
  Simulated time: 1.0 s; 1 workers
  Run time: 0.23 s
   TPO : 0.000 Hz
   TVL : 0.000 Hz
   S1 : 0.000 Hz
   S2 : 0.000 Hz
   cM1 : 0.000 Hz
   M2 : 0.000 Hz
   OC : 0.000 Hz
Saving output as ../data/batch_2023-12-08/batch_2023-12-08_4_4_0_16_data.pkl ... 
Finished saving!
  Done; saving time = 0.31 s.
Preparing LFP data...
Plotting LFP time series...
  There was an exception in plotLFPTimeSeries(): 
    zero-size array to reduction operation maximum which has no identity 
    (<class 'ValueError'>, ValueError('zero-size array to reduction operation maximum which has no identity'), <traceback object at 0x293a24740>)
Preparing spike data...
No spikes available to plot raster
Plotting raster...
  There was an exception in plotRaster(): 
    cannot access local variable 'spkInds' where it is not associated with a value 
    (<class 'UnboundLocalError'>, UnboundLocalError("cannot access local variable 'spkInds' where it is not associated with a value"), <traceback object at 0x16fe65180>)
Plotting recorded cell traces ... trace
  Done; plotting time = 0.05 s

Total time = 0.74 s
